docs/openai.md

```diff
@@ -27,6 +27,8 @@ chat_completion = client.chat.completions.create(
     ],
     model='llama3',
 )
+
```

Comment:
Awesome work adding the docs! Let's make this a separate PR we can merge once the latest Ollama release includes it